### DISCUSSION TITLE
Started the development environment but it gives an error

### DISCUSSION THREAD

**Discussion by JonasTheGamer** [getredash/redash#6880] [Q&A] [answered]
Here I am again haha. I tried following the local dev environment guide and everything works. However, when I open the website at localhost:5000 it gives me this error:
![image](https://github.com/getredash/redash/assets/38383712/ef73b6a9-c9ca-4fb3-8f01-6a08fb04d9fa)

What would I be doing wrong?

**Comment by justinclift** [comment]
@AndrewChubatiuk Would you be ok to assist here?  I haven't yet tried out the development pieces since your PRs were merged.

**Reply by AndrewChubatiuk** [reply]
Now by default it spins up local setup in a dev mode (without gunicorn), but there are some issues, which possibly should be fixed after migration to the latest flask + sqlalchemy
[in PR ](https://github.com/getredash/redash/pull/6882) enabled gunicorn for a local setup by default

**Reply by justinclift** [reply]
Ok, that one's been merged too.

**Reply by justinclift** [reply]
@AndrewChubatiuk Seeing lots of CI failures now, after what seems like the merge of #6877.

I'm getting tempted to just revert `master` back to the working state it was in before #6674.

If we do that, then things will be working again and we can look at the subsequent changes again far more carefully.

**Reply by AndrewChubatiuk** [reply]
CI pipelines should be fine now

**Reply by justinclift** [reply]
~@AndrewChubatiuk Can you please take a look at those errors?  We should try to get this fixed in the next few hours. :smile:~

**Reply by justinclift** [reply]
Ahhh cool.  I'll try re-running the CI then.

**Reply by justinclift** [reply]
Looks like they're passing now after #6882.  Cool.  Thanks heaps. :smile:

**Comment by justinclift** [comment]
@JonasTheGamer Sorry for the weirdness here.  We merged a rather large patch set yesterday that seems to need a bit of extra hammering into shape. :grin:

**Reply by JonasTheGamer** [reply]
Haha no worries. Thanks! Let me see if I can spin it up now (I'll just redo the setup)

**Reply by justinclift** [reply, marked as answer]
If you revert the `master` branch back to [this commit](https://github.com/getredash/redash/commit/24dec192eed8c95560ef20f6765d96c67a5bdbbf) then things should work for you.

It's that stuff after that which is causing issues.  We should have that sorted out tonight one way or another. :smile:

**Reply by JonasTheGamer** [reply]
Works like a charm, thanks! :-)
